### PR TITLE
S5 us16/feat modal get location

### PR DESCRIPTION
**Pull request by marcDjango**

### CODE REVIEW

**Review by QuentinVay — APPROVED** [review]

File: frontend/src/components/user/LoginPage/LoginPage.jsx [hunk 1]
```diff
@@ -7,12 +7,13 @@ import "./login.scss";
 import RegistrationForm from "../RegistrationPage/RegistrationForm";
 
 function LoginPage() {
+  // const [isAdmin, setIsAdmin] = useState(false);
```
Comment (ArthurHtbk) [inline frontend/src/components/user/LoginPage/LoginPage.jsx]: 😱

File: frontend/src/components/user/LoginPage/LoginPage.jsx [hunk 2]
```diff
@@ -40,13 +41,22 @@ function LoginPage() {
...
       });
-      // Redirection vers la page de connexion si la création réussit
       const user = await response.json();
       if (response.status === 200) {
-        setAuth(user.user);
-        localStorage.setItem("user", JSON.stringify(user.user));
-        localStorage.setItem("token", user.token);
-        navigate("/");
+        if (user.user.is_admin === 1) {
+          // setIsAdmin(true);
```
Comment (ArthurHtbk) [inline frontend/src/components/user/LoginPage/LoginPage.jsx]: 😱

File: frontend/src/components/user/LoginPage/LoginPage.jsx [hunk 3]
```diff
@@ -60,33 +70,33 @@ function LoginPage() {
       setIsSubmit(true);
     }
   };
+  // useEffect(() => {
```
Comment (ArthurHtbk) [inline frontend/src/components/user/LoginPage/LoginPage.jsx]: 😱

**Review by ArthurHtbk — CHANGES_REQUESTED** [review]

File: frontend/src/components/profile/car/CardCar.jsx [hunk 4]
```diff
@@ -1,14 +1,15 @@
-import { useState } from "react";
+/* eslint-disable no-await-in-loop */
```
Comment (ArthurHtbk) [inline frontend/src/components/profile/car/CardCar.jsx]: 😱

File: frontend/src/components/profile/car/CardCar.jsx [hunk 5]
```diff
@@ -17,8 +18,27 @@ function CardCar() {
...
   const dataCars = useLoaderData();
   const dataProps = null;
+
+  const [loadedImages, setLoadedImages] = useState({});
+
+  useEffect(() => {
+    const loadImages = async () => {
+      const loaded = {};
+      for (const key of Object.keys(images)) {
+        loaded[key] = (await images[key]).default;
```
Comment (ArthurHtbk) [inline frontend/src/components/profile/car/CardCar.jsx]: 😱

**Review by ArthurHtbk — CHANGES_REQUESTED** [review]

File: frontend/src/components/user/LoginPage/LoginPage.jsx [hunk 6]
```diff
@@ -12,7 +12,7 @@ function LoginPage() {
   const [isErrors, setIsErrors] = useState(null);
   const [isSubmit, setIsSubmit] = useState(false);
   const [isSignupModal, setIsSignupModal] = useState(false);
-
+  console.info(auth);
```
Comment (ArthurHtbk) [inline frontend/src/components/user/LoginPage/LoginPage.jsx]: Nope!

**Review by ArthurHtbk — CHANGES_REQUESTED** [review]

**Review by ArthurHtbk — APPROVED** [review]

**Pull request merged**